### ISSUE TITLE
ULS UNITS MANAGEMENT - Embedded video fixes

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/uls#462]
Management wants to be able to embed videos on their site. This can be done with the ACF Video field. Need to add a field to be able to paste in a transcript/audio description/text accompaniment when any video is added.

There is also an issue with the iframe not showing a title attribute, but looking at the code on the front end, the video does have a title attribute. Will need to do some further research.

"YouTube iframe should not use html presentation attributes, change to CSS in style attribute" - also not sure about this one because this is being generated by ACF with `echo the_field('video')` and not sure if this is coming from ACF or from Youtube.

**Comment by mhk33** [comment]
I worked with Neely on Kaltura videos before, so I know she knows this, but just a reminder that videos should have edited captions. If the videos go into Kaltura and we use the Kaltura embed code, it may be possible to show a dynamic, searchable transcript along with the video. For example:
https://media.library.cornell.edu/media/Adam+LevineA+American+Insecurity/1_ftxohfmw/5877591
